### DISCUSSION TITLE
Code cleanup strategy

### DISCUSSION THREAD

**Discussion by gnl42** [eclipse-mylyn/org.eclipse.mylyn#311] [Polls]
With #200 now in place, we should really clean up the code base to take advantage of the updated code style settings

**Comment by BeckerFrank** [comment]
For the Gitlab Connector we already have https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/310

**Comment by gnl42** [comment]
Which makes sense since you are actively working on it so no chance of merge problems.

The rest of the code base is pretty static at the moment (as far as I can tell) so there is really no danger of merge conflicts messing up anyone 's local changes

**Comment by gnl42** [comment]
Done with #375, ##376, #377, #378, #379. #380, #381, #382, #383, #384, #385
